src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -185,6 +185,55 @@ export const exchangeStitchingEnvironment = ({
     },
   }
 
+  const paymentMethodResolver = {
+    fragment: gql`
+      fragment CommerceOrderPaymentMethod on CommerceOrder {
+        creditCardId
+        bankAccountId
+        paymentMethod
+      }
+    `,
+
+    resolve: async (parent, _args, context, info) => {
+      const { creditCardId, bankAccountId, paymentMethod } = parent
+
+      if (paymentMethod === "credit card") {
+        // assert creditCardId exists?
+        return info.mergeInfo.delegateToSchema({
+          schema: localSchema,
+          operation: "query",
+          fieldName: "creditCard",
+          args: {
+            id: creditCardId,
+          },
+          context,
+          info,
+          transforms: exchangeSchema.transforms,
+        })
+        // note: us_bank_account with underscore *is* the current value from exchange
+      } else if (paymentMethod === "us_bank_account") {
```

Comment:
Can we rename this string on exchange to be consistent with the space-cased strings for credit card and wire transfer?